src/mapper/cmd/main.go

```diff
@@ -79,11 +83,23 @@ func main() {
 		logrus.Error(intentHolderCfg)
 		os.Exit(1)
 	}
+
+	cloudConfig := clouduploader.ConfigFromViper()
 	intentsHolder := resolvers.NewIntentsHolder(mgr.GetClient(), intentHolderCfg)
+	cloudClient := clouduploader.NewCloudUploader(intentsHolder, cloudConfig, cloudapi.CloudAPIImpl{})
```

Comment:
If you initiate an Impl struct on your own, you are doing something wrong. Add a `NewCloudClient()` method